### ISSUE TITLE
Use Prettier for SFC <script> tag content formatting

### ISSUE THREAD

**Issue by onlyurei** [vuejs/eslint-plugin-vue#267]
**Tell us about your environment**

* **ESLint Version:**
4.12.0
* **eslint-plugin-vue Version:**
4.0.0-beta.0
* **Node Version:**
8.6.0

**Please show your full configuration:**

`<projectRoot>/.eslint.json`
```
{
  "root": true,
  "extends": [
    "prettier",
    "plugin:vue/recommended"
  ],
  "plugins": ["html"]
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue





```

**What did you expect to happen?**
Prettier should lint the content inside `<script>` tag of Vue SFCs, and eslint-plugin-vue should lint the content inside `<template>` tag of Vue SFCs.

**What actually happened? Please include the actual, raw output from ESLint.**
Content inside `<template>` tag of Vue SFCs not linted by eslint-plugin-vue.

Related Prettier issues:
https://github.com/prettier/prettier/issues/2097
https://github.com/prettier/prettier/issues/1674

**Comment by mysticatea** [comment]
Thank you for this issue.

I think that this is a duplicate of #259 and #261.

**Comment by michalsnik** [comment]
I agree with @mysticatea . Please feel free to reopen this issue if you'll continue to have problems. But it's well explained in mentioned issues ☝️

**Comment by onlyurei** [comment]
@mysticatea @michalsnik looks like https://github.com/prettier/prettier/pull/3563 added "proper support for Vue Single File Components (SFC)", I'll play around the combination of using both `eslint-plugin-vue` and `eslint-plugin-prettier` to lint/fix Vue SFC when that PR is [published](https://github.com/prettier/prettier/issues/3603#issue-285078380). 

However I also saw this [comment](https://github.com/prettier/prettier/issues/2097#issuecomment-354149800):

> @oligot However, prettier still doesn't work witheslint-plugin-vue@next, which has more concrete style guild. Your test project is based on eslint plugin vue 2.0.0.

**Comment by onlyurei** [comment]
@mysticatea @michalsnik looks like https://github.com/prettier/prettier/pull/3563 added "proper support for Vue Single File Components (SFC)", I'll play around the combination of using both `eslint-plugin-vue` and `eslint-plugin-prettier` to lint/fix Vue SFC when that PR is [published](https://github.com/prettier/prettier/issues/3603#issue-285078380). 

However I also saw this https://github.com/prettier/prettier/issues/2097#issuecomment-354149800:

> @oligot However, prettier still doesn't work witheslint-plugin-vue@next, which has more concrete style guild. Your test project is based on eslint plugin vue 2.0.0.
